docs/resources/Channel.md

```diff
@@ -869,6 +884,16 @@ Create a new [invite](#DOCS_RESOURCES_INVITE/invite-object) object for the chann
 
 Delete a channel permission overwrite for a user or role in a channel. Only usable for guild channels. Requires the `MANAGE_ROLES` permission. Returns a 204 empty response on success. For more information about permissions, see [permissions](#DOCS_TOPICS_PERMISSIONS/permissions)
 
+## Follow News Channel % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/followers
+
+Follow a News Channel to send messages to a target channel. Requires the `MANAGE_WEBHOOKS` permission in the target channel. Returns a [followed channel](#DOCS_RESOURCES_CHANNEL/followed-channel-object) object.
+
+###### JSON Params
+
+| Field              | Type   | Description          |
+| ------------------ | ------ | -------------------- |
+| webhook_channel_id | string | id of target channel
```

Comment:
You're missing a `|` here at the end of this line and the type of a snowflake id should be `snowflake` here as well

Reply:
Will get to these tomorrow morning